keygen/tests/test_logic.py

```diff
@@ -181,3 +183,14 @@ def test_tmpfiles_errors(self, popen, user_exists):
                 logic.create_new_key(app, TEST_NAME, TEST_EMAIL, TEST_KEYLENGTH)
 
             assert not popen.called
+
+    def test_validate_name_email(self, _popen, _user_exists):
+        assert logic.validate_name_email("frostyx#foo@copr.fedorahosted.org")
+        assert logic.validate_name_email("@copr#foo@copr.fedorahosted.org")
```

Comment:
Just note: https://bugzilla.redhat.com/show_bug.cgi?id=1813244 and #1320

Perhaps we should add a small TODO at least?